README.md

```diff
@@ -247,6 +247,8 @@ If you are reading this it looks like you are looking to add an egg to your serv
 ## [Software](/software/)
 ### Code Server
 * [Code-Server](/software/code-server)
+### Fosscord-server
+* [Fosscord-server](/software/fosscord-server)
```

Comment:
Please follow the alphabetical listing, which would put this after Elasticsearch

Reply:
> Please follow the alphabetical listing, which would put this after Elasticsearch

Kk